cmd/gardenlet/app/migration.go

```diff
@@ -153,3 +159,36 @@ func deleteOrphanedValiVPAs(ctx context.Context, log logr.Logger, c client.Clien
 
 	return flow.Parallel(fns...)(ctx)
 }
+
+// TODO(MichaelEischer): Remove this function after v1.101 has been released
+func createOSCHashMigrationSecret(ctx context.Context, seedClient client.Client) error {
+	namespaceList := &corev1.NamespaceList{}
+	if err := seedClient.List(ctx, namespaceList, client.MatchingLabels(map[string]string{v1beta1constants.GardenRole: v1beta1constants.GardenRoleShoot})); err != nil {
+		return err
+	}
+
+	var tasks []flow.TaskFn
+	for _, ns := range namespaceList.Items {
+		if ns.DeletionTimestamp != nil || ns.Status.Phase == corev1.NamespaceTerminating {
+			continue
+		}
+		namespace := ns
```

Comment:
I think such line is no longer needed now that we have Go 1.22

Reply:
You're right, I've removed it.